polyglot/piranha/src/utilities/unit_tests/tree_sitter_utilities_test.rs

```diff
@@ -128,140 +120,6 @@ fn test_get_all_matches_for_query_negative() {
   assert!(matches.is_empty());
 }
 
-#[test]
-fn test_satisfies_constraints_positive() {
```

Comment:
move to `source_code_unit_tests.rs` because we moved the `is_satsified` function to `source_code_unit.rs`